### PR TITLE
Barak/imports parser package

### PR DESCRIPTION
**Pull request by barak007**

### CODE REVIEW

File: packages/imports-parser/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# @tokey/imports-parser
+
+
+Parser for esm like import syntax with support for "functional" tagged imports
```
Comment (tomrav) [inline packages/imports-parser/README.md:4]:
```suggestion
Parser for `esm` like import syntax with support for "functional" tagged imports
```

File: packages/imports-parser/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@tokey/imports-parser",
+  "description": "parser for esm like imports",
+  "version": "0.0.1",
```
Comment (tomrav) [inline packages/imports-parser/package.json:4]: if this is the "production" version, consider using `v1.0.0`

**Review by tomrav — APPROVED** [review]
Looks good

File: packages/imports-parser/package.json (hunk 2 above)
Comment (barak007) [reply packages/imports-parser/package.json:4]: followed by the selector-parser versioning...not sure we want to commit to not breaking. 
WDTY?

Comment (tomrav) [reply packages/imports-parser/package.json:4]: I'm not scared of major versions, but if you see it making breaking changes again several times in the near future then okay.

**Pull request merged**